### DISCUSSION TITLE
How do you update Cosmos?

### DISCUSSION THREAD

**Discussion by EnZon3** [CosmosOS/Cosmos#2139] [Q&A]
I want to update to the latest userkit but idk how to, can anyone help me?

**Comment by zarlo** [comment]
you just need to download the installer and run it (and for devkit you just repull all the repos and rerun the installer)

**Reply by EnZon3** [reply]
Do you just like overwrite the existing userkit folder?

**Reply by EnZon3** [reply]
because it is stuck on 
![image](https://user-images.githubusercontent.com/37979038/155900147-7fb10a2d-d7a6-4dc5-9b6f-82da3ccbfc63.png)

**Reply by abjdev** [reply]
im removing cosmos and reinstalling. that should work without troubles
